core/src/main/java/com/linecorp/armeria/server/DefaultUnhandledExceptionsReporter.java

```diff
@@ -50,7 +55,7 @@ final class DefaultUnhandledExceptionsReporter implements UnhandledExceptionsRep
     @Nullable
     private ScheduledFuture<?> reportingTaskFuture;
     @Nullable
-    private Throwable thrownException;
+    private Map<Throwable, Pair<LongAdder, ServiceRequestContext>> thrownExceptions;
```

Comment:
```
    // akka
    implementation libs.akka.actor.typed.v213
```
```java
private static class ThrowableEntry {
  LongAdder adder;
  ServiceRequestContext ctx;
}

private Map<Throwable, ThrowableEntry> thrownExceptions;
Comment

```
oh instead of import `Pair` on `akka`, can we add some class like `ThrowableEntry`?
then we don't need to import akka 😄

Reply:
We don't want to add an additional dependency for this. Should we use `Map.Entry` or a custom class as suggested by @injae-kim instead?